ext/tidy/tidy.c

```diff
@@ -30,7 +30,7 @@
 #include "php_ini.h"
 #include "ext/standard/info.h"
 
-#include "tidy.h"
+#include TIDY_INC_H
```

Comment:
Unneeded if TIDY_INCDIR is in the include path.

Reply:
This looks somewhat weird. The way we'd usually do this is check HAVE_TIDY_H and include this header and check HAVE_TIDYP_H and include that header.